### PR TITLE
feat: upload data & delete sample mode

### PR DESCRIPTION
**Pull request by zhouzilong2020**
We now push our data to the repo, so there is no need for a sample mode.

### CODE REVIEW

File: frontend/stores/dbInstance.ts [hunk 1]
```diff
@@ -75,8 +75,9 @@ export default defineStore("dbInstance", {
       if (config.public.mode === "sample") {
         sourceFileName = "sample.json";
       }
+      console.log(`loading [${config.public.mode}] data(${sourceFileName})`);
```
Comment (LiuJi-Jim) [inline frontend/stores/dbInstance.ts:78]: Debug codes should be removed.

**Review by LiuJi-Jim — COMMENTED** [review]
LG otherwise

**Comment by zhouzilong2020** [comment]
PTAL

**Review by LiuJi-Jim — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**